docs/metrics.md

```diff
@@ -19,6 +19,7 @@ Other available metrics can found at [Vert.x Dropwizard Metrics](https://vertx.i
 - `request_time` - timer tracking how long did it take for Prebid Server to serve a request
 - `imps_requested` - number if impressions requested
 - `requests.(ok|badinput|err|networkerr).(openrtb2-web|openrtb-app|amp|legacy)` - number of requests broken down by status and type
+- `connection_accept_errors` - number of requests broken down by errors occurred while establishing HTTP connection
```

Comment:
Let's change the description to "number of errors occurred while establishing HTTP connection"